Interpreter.py

```diff
@@ -40,7 +41,7 @@ def brainfuck(program):
             data_pointer -= 1
         elif command == '+':
             data[data_pointer] = (data.get(data_pointer, 0) + 1)
-            if data[data_pointer] == 256:
+            if data[data_pointer] == (int(255 / 8) * int(bits)) + 1:
                 data[data_pointer] = 0
         elif command == '-':
```

Comment:
Please use the correct amount of bits here as well. Currently it is hard-coded to 255